admin/templates/users/user.html

```diff
@@ -32,23 +32,39 @@
                 <a href="{% url 'desk:customer' user.id %}" data-toggle="modal" data-target="#deskModal" class="btn btn-default">Desk profile</a>
                 <a href="{% url 'desk:user_cases' user.id %}" data-toggle="modal" data-target="#caseModal" class="btn btn-default">Desk cases</a>
                 {% endif %}
+                <a href="{% url 'users:reset_password' user.id %}"
+                   data-toggle="modal" data-target="#resetModal"
+                   class="btn btn-primary">
+                    Send reset
+                </a>
+                <a href="{% url 'users:reindex-elastic-user' guid=user.id %}" data-toggle="modal" data-target="#confirmReindexElasticUser" class="btn btn-success">Elastic Reindex</a>
+
+
             </div>
         </div>
-        <div class="col-md-4 form-group">
-            {% if not user.disabled %}
-                <a href="{% url 'users:disable' user.id %}" data-toggle="modal" data-target="#disableModal" class="btn btn-danger">Disable account</a>
-            {% elif 'spam_confirmed' not in user.system_tags %}
-                <form method="post" action="{% url 'users:reactivate' user.id %}">
-                    {% csrf_token %}
-                    <input class="btn btn-success" type="submit" value="Reactivate account"/>
-                </form>
-            {% endif %}
-            {% if not user.disabled or 'spam_flagged' in user.system_tags %}
-                <a href="{% url 'users:spam_disable' user.id %}" data-toggle="modal" data-target="#disableSpamModal" class="btn btn-danger">Disable Spam account</a>
-            {% elif 'spam_confirmed' in user.system_tags %}
-                <a href="{% url 'users:ham_enable' user.id %}" data-toggle="modal" data-target="#enableHamModal" class="btn btn-success">Re-enable Ham account</a>
-            {% endif %}
-            <a href="{% url 'users:reindex-elastic-user' guid=user.id %}" data-toggle="modal" data-target="#confirmReindexElasticUser" class="btn btn-success">Elastic Reindex</a>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <div class="btn-group padded" role="group">
+                {% if not user.disabled %}
+                    {% if user.requested_deactivation %}
+                        <a href="{% url 'users:disable' user.id %}" data-toggle="modal" data-target="#disableModal" class="btn btn-danger">Requested account deactivation</a>
```

Comment:
@Johnetordoff is this right? It's the same as the button if the user hasn't requested deactivation...

Reply:
Oh except it's not disabled. Is this how the admin is suppose to cancel the request or did I misunderstand the feature?